pattern-1/bosh-release/jobs/wso2apim_analytics/spec

```diff
@@ -32,32 +32,38 @@ properties:
   wso2apim_analytics.address:
     description: wso2apim address
 
-  wso2apim.carbon_db.hostname:
-    description: WSO2 APIM MySQL server hostname
-  wso2apim.carbon_db.db_name:
-    description: WSO2 APIM MySQL database name
+  wso2apim.carbon_db.jdbc_url:
+    description: WSO2 Carbon JDBC URL
```

Comment:
Shall we change the description to "WSO2 Carbon database JDBC URL"?

Reply:
fixed https://github.com/wso2/pivotal-cf-apim/pull/52/commits/35c6554ff64c8f44c531e2ed5ac7534cb6a2717d